Java/Interfaces/Defining_And_Implementation_Of_Interfaces/Defining_and_implementation_of_interface.md

```diff
@@ -0,0 +1,88 @@
+# Defining Interface and it's Implementation 
+---------
+## What is an Interface?
+An interface is quite similar to some other class, but it has got some method that have no body or in other words no implementation and some of the constant values. So in general interface is just a blueprint of a class which has static constant and abstract methods .
```

Comment:
`other class` -> `other classes`
`some method` -> `some methods`
`class which has static constant ` -> `class that has static constants `